src/main/java/ca/gc/tri_agency/granting_data/service/impl/GrantingCapabilityServiceImpl.java

```diff
@@ -45,7 +46,8 @@ public GrantingCapability saveGrantingCapability(GrantingCapability gc) {
 
 	@Override
 	@AdminOnly
-	public void deleteGrantingCapability(Long id) {
+	@Transactional
+	public void deleteGrantingCapabilityById(Long id) {
 		gcRepo.delete(findGrantingCapabilityById(id));
```

Comment:
I made this entire method transactional because this way only 1 transaction is opened and closed whereas before there would be 2 transactions: one for findGrantingCabapilityById and another for delete